monkey/infection_monkey/network_scanning/scan_target_generator.py

```diff
@@ -68,17 +72,17 @@ def _get_ips_from_subnets_to_scan(subnets_to_scan: List[str]) -> List[NetworkAdd
     return _get_ips_from_ranges_to_scan(network_ranges)
 
 
-def _get_ips_from_ranges_to_scan(network_ranges: List[NetworkRange]) -> List[NetworkAddress]:
+def _get_ips_from_ranges_to_scan(network_ranges: Iterable[NetworkRange]) -> List[NetworkAddress]:
     scan_targets = []
 
     for _range in network_ranges:
         scan_targets.extend(_range_to_addresses(_range))
     return scan_targets
 
 
-def _get_ips_to_scan_from_local_interface(
-    interfaces: List[IPv4Interface],
-) -> List[NetworkAddress]:
+def _get_ips_to_scan_from_interface(
+    interfaces: Sequence[IPv4Interface],
+) -> Sequence[NetworkAddress]:
```

Comment:
Mypy doesn't complain here (and in other places) about this being a general return type?
https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides

Reply:
Nope! In fact, the linked document explains why :-)